xslt/dita2html.xsl

```diff
@@ -40,6 +40,10 @@
       <xsl:value-of select="normalize-space($commit)"/>
       <xsl:text>"&#xA;</xsl:text>
     </xsl:if>
+    <xsl:if test="contains(/*/@outputclass, 'generated') or contains(/*/title[1]/@outputclass, 'generated')">
```

Comment:
Don't match against plain element names like `title`, use `*[contains(@class, ' topic/title ')]`.

`@outputclass` can contain multiple values, thus use `tokenize(@outputclass, '\s+') = 'generated'` instead of `contains()`.